src/pages/PokemonDetailsPage/PokemonDetailsPage.css

```diff
@@ -105,3 +101,65 @@
 .bold {
   font-weight: bold;
 }
+
+@media screen and (max-width: 440px) {
+  .button-and-name {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  }
+
+  .copy-to-clipboard {
+    margin: 0;
+    grid-column-start: 2;
+  }
+
+  .stats-title {
+    grid-column-start: 4;
+  }
+
+  .display-section {
+    max-width: 430px;
+    flex-grow: 1;
+  }
+  .stats-window {
+    margin: 0;
+    border-radius: 0;
+    height: 662px;
+    width: 430px;
+  }
+
+  .image-window {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .pokemon-image {
+    grid-column-start: 2;
+  }
+
+  .image-arrow {
+    grid-column-start: 3;
+    padding: 0;
+    font-size: 2.5rem;
+  }
+
+  .text-and-image-wrapper {
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+  }
+  .pokemon-image {
+    width: 280px;
+    height: 280px;
```

Comment:
Similar comment to before. 

I would consider any time you have a hard pixel value to be a code smell, and you can almost always achieve better results by using relative values or css grid.